completions/just.bash

```diff
@@ -26,6 +26,13 @@ _just() {
                     return 0
                 elif [[ ${COMP_CWORD} -eq 1 ]]; then
                     local recipes=$(just --summary --color never 2> /dev/null)
+
+                    if echo "${cur}" | grep -qF '/'; then
+                        local path_prefix=$(echo "${cur}" | sed 's/[/][^/]*$/\//')
+                        local recipes=$(just --summary --color never 2> /dev/null -- "${path_prefix}")
```

Comment:
I think summary never colors its output:

```suggestion
                        local recipes=$(just --summary 2> /dev/null -- "${path_prefix}")
```

Reply:
FWIW, that was already there before. If it's not needed I'll do the change, of course.